prover/lib/compressor/blob/blob.go

```diff
@@ -25,27 +23,7 @@ func GetVersion(blob []byte) uint16 {
 	return 0
 }
 
-// GetRepoRootPath assumes that current working directory is within the repo
-func GetRepoRootPath() (string, error) {
-	wd, err := os.Getwd()
-	if err != nil {
-		return "", err
-	}
-	const repoName = "linea-monorepo"
-	i := strings.LastIndex(wd, repoName)
-	if i == -1 {
-		return "", errors.New("could not find repo root")
-	}
-	i += len(repoName)
-	return wd[:i], nil
-}
-
-func GetDict() ([]byte, error) {
-	repoRoot, err := GetRepoRootPath()
-	if err != nil {
-		return nil, err
-	}
-	dictPath := filepath.Join(repoRoot, "prover/lib/compressor/compressor_dict.bin")
+func GetDict(dictPath string) ([]byte, error) {
```

Comment:
The goal is here to move any function with a dependency on [GetRepoRootPath] confined to testing package